payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentLauncherViewModel.kt

```diff
@@ -1,32 +1,89 @@
 package com.stripe.android.payments.paymentlauncher
 
+import android.content.Context
+import androidx.activity.result.ActivityResultCaller
+import androidx.activity.result.ActivityResultLauncher
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
+import com.stripe.android.StripeIntentResult
+import com.stripe.android.auth.PaymentBrowserAuthContract
+import com.stripe.android.exception.APIException
 import com.stripe.android.model.ConfirmStripeIntentParams
+import com.stripe.android.model.StripeIntent
+import com.stripe.android.view.AuthActivityStarterHost
 
 /**
  * WIP - view model for PaymentLauncherHostActivity
  */
+
 internal class PaymentLauncherViewModel : ViewModel() {
+    internal lateinit var paymentBrowserAuthLauncher:
+        ActivityResultLauncher<PaymentBrowserAuthContract.Args>
```

Comment:
Can we avoid this pattern? We should keep Activity-scoped objects out of the ViewModel if possible.